src/hammer-vlsi/defaults.yml

```diff
@@ -454,6 +454,21 @@ par.inputs:
   # type: bool
   gds_merge: false
 
+  # Physical only cells mode.
+  # type: str
+  # The list of physical only cells are cells which don't have a logic function.
+  # They won't have a spice circuit for instance.
+  # This is currently only used for what cells to exclude from the LVS netlist, which
```

Comment:
Does this include filler/decap cells? May be related to #180 

Also, how does it affect LVS? Does it mean that if a physical-only cell causes a short, it won't be detected?

Reply:
Well, filler and decap cells can and should be treated differently here. Filler cells we remove- they have no LVS-able devices in them. Decap cells do. LVS should still detect a short.